application.json

```diff
@@ -55,6 +55,8 @@
         "@fabric-ds",
         "@metrics",
         "@warp-ds",
+        "@m10s",
```

Comment:
If those prefixes should be treated as other internal dependencies, should we maybe add them to the excludePackagePrefixes lists in the other two rules above, too?